src/test/java/spoon/support/compiler/classpath/ComputeClasspathTest.java

```diff
@@ -48,32 +47,8 @@ public SpoonCompiler createCompiler(Factory factory) {
 	}
 
 	@Test
-	public void testTemplateClasspath()
```

Comment:
This test is no more necessary because my refactoring unifies management of source and template classpaths.